### PR TITLE
[Docs] Add Paperless Mobile app to docs

### PR DESCRIPTION
**Pull request by astubenbord**
## Proposed change

Add [Paperless Mobile](https://github.com/astubenbord/paperless-mobile) to the list of possible mobile upload options.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please explain)

Update of documentation.

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -108,6 +108,8 @@ Furthermore, there is the [Paperless
 App](https://github.com/bauerj/paperless_app) as well, which not only
 has document upload, but also document browsing and download features.
 
+Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), a feature rich Paperless mobile client supporting document upload, scanning,  management of labels and much more.
```
Comment (shamoon) [inline docs/usage.md:111]:
```suggestion
Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), an Android app that supports document upload, scanning,  management of labels and more.
```

**Review by shamoon — CHANGES_REQUESTED** [review]
Thanks, cool looking app. I just shortened the text a bit for parity and added the fact that it's android

**Comment by shamoon** [comment]
- Ps. You also need to address the linting failure, we use
`pre-commit` but could easily do manually for the 1 line

File: docs/usage.md [hunk 2]
```diff
@@ -108,6 +108,8 @@ Furthermore, there is the [Paperless
 App](https://github.com/bauerj/paperless_app) as well, which not only
 has document upload, but also document browsing and download features.
 
+Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), an Android app that supports document upload, scanning,  management of labels and more.
```
Comment (shamoon) [inline docs/usage.md:111]:
```suggestion
Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), an Android app that supports document upload, scanning, management of labels and more.
```

**Review by shamoon — APPROVED** [review]

**Review by stumpylog — APPROVED** [review]

**Pull request merged**